packages/cds-odata-annotation-converter/test/printer/csdl-to-cds.test.ts

```diff
@@ -0,0 +1,2399 @@
+                                value: 'UI.ParameterDefaultValue',
+                                valueRange: {
+                                    start: { line: 57, character: 12 },
+                                    end: { line: 57, character: 36 }
+                                }
+                            },
+                            Qualifier: {
+                                type: 'attribute',
+                                name: 'Qualifier',
+                                nameRange: null,
+                                value: 'paramAfter',
+                                valueRange: {
+                                    start: { line: 57, character: 38 },
+                                    end: { line: 57, character: 48 }
+                                }
+                            }
+                        },
+                        range: {
+                            start: { line: 57, character: 12 },
+                            end: { line: 57, character: 55 }
+                        },
+                        contentRange: {
+                            start: { line: 57, character: 50 },
+                            end: { line: 57, character: 55 }
+                        },
+                        content: [
+                            {
+                                type: 'element',
+                                name: 'Bool',
+
+                                range: {
+                                    start: { line: 57, character: 50 },
+                                    end: { line: 57, character: 55 }
+                                },
+                                contentRange: {
+                                    start: { line: 57, character: 50 },
+                                    end: { line: 57, character: 55 }
+                                },
+                                content: [
+                                    {
+                                        type: 'text',
+                                        text: 'false',
+                                        range: {
+                                            start: { line: 57, character: 50 },
+                                            end: { line: 57, character: 55 }
+                                        }
+                                    }
+                                ],
+                                attributes: {}
+                            }
+                        ]
+                    }
+                ]
+            };
+
+            // act
+            const result = printTarget(target);
+
+            // assert
+            expect(result).toMatchSnapshot();
+        });
+
+        test('Annotating unbound Action/Function parameter', () => {
+            // arrange
+            const target: Target = {
+                type: 'target',
+                name: 'AdminService.cancelOrder()/orderID',
+                range: {
```

Comment:
Ranges are optional and not used. We should remove them to make the test a bit more compact.

Reply:
remmoved.